composer.json

```diff
@@ -34,22 +33,24 @@
         "colinmollenhour/cache-backend-file": "1.4",
         "colinmollenhour/cache-backend-redis": "1.10.2",
         "colinmollenhour/credis": "1.6",
-        "colinmollenhour/php-redis-session-abstract": "~1.2.2",
-        "composer/composer": "1.4.1",
-        "magento/composer": "~1.2.0",
+        "colinmollenhour/php-redis-session-abstract": "~1.3.8",
+        "composer/composer": "~1.6.0",
+        "friendsofphp/php-cs-fixer": "~2.10.0",
+        "magento/composer": "1.3.0.x-dev",
         "magento/magento-composer-installer": ">=0.1.11",
         "magento/zendframework1": "~1.13.0",
         "monolog/monolog": "^1.17",
         "oyejorge/less.php": "~1.7.0",
         "pelago/emogrifier": "^2.0.0",
         "phpseclib/phpseclib": "2.0.*",
         "ramsey/uuid": "3.6.1",
-        "sjparkinson/static-review": "~4.1",
-        "symfony/console": "~2.3, !=2.7.0",
-        "symfony/event-dispatcher": "~2.1",
-        "symfony/process": "~2.1",
+        "sebastian/phpcpd": "~3.0.0",
```

Comment:
Why is it moved to `require` from `require-dev`?

Reply:
No reason. Moved back to `require-dev`.